### PR TITLE
Footer Component Added

### PR DESCRIPTION
**Pull request by Chandan-SM**
1. Footer Component Created.
2. Minor CSS bug fix in the home page body. 
3. Footer Made completely Responsive.

Screenshot 2023-05-18 at 4 59 25 PM

### CODE REVIEW

**Review by PatelNeelMahesh — APPROVED** [review]
Good work, @Chandan-Gowda-Kalvium APKInfo looks even better with the content rich footer.

File: Client/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <div style={{marginTop:"87vh", borderTop:"3px solid green"}}>
+      <footer style={{display:"flex", flexDirection:"column", marginTop:"", justifyContent:"center", alignItems:"center", backgroundColor:"black"}}>
+        <div className="footer" style={{backgroundColor:"black",display:"flex",flexDirection:"row", justifyContent:"center", alignItems:"center"}}>
+        <div className="logo-footer" style={{width:"50%", backgroundColor:"black"}}>
+          <div className="logo-text" style={{backgroundColor:"black",display:"flex",flexDirection:"column", justifyContent:"center", alignItems:"center"}}>
+            <img src={logo} alt=""  style={{backgroundColor:"black", width:"20%", marginTop:"4vh", marginBottom:"2vh"}}/>
+            <p style={{marginRight:"0%",width:"80%" ,textAlign:"center", marginTop:"1vh",fontSize:"large", color:"white", backgroundColor:"transparent"}}>A website which provides comprehensive information about any uploaded APK file, including details about the app's package, version, permissions, and signatures. We also offer security assessments, code analysis, user reviews, and FAQs to help you make informed decisions about installing and using the APK.</p>
```
Comment (BalajiSriraman) [inline Client/src/components/Footer/Footer.jsx:16]: kindly follow text indentation or code formatting!

File: Client/src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                <li style={{color:"white", backgroundColor:"transparent", marginBottom:".6vh"}}><Link className="links" to={"/faq"} onClick={() => window.scrollTo(0, 0)} style={{fontSize:"20px"}}>FAQ</Link></li>
+                <li style={{color:"white", backgroundColor:"transparent", marginBottom:".6vh"}}><Link className="links" to={"/t&c"} onClick={() => window.scrollTo(0, 0)} style={{fontSize:"20px"}}>T&C</Link></li>
+                <li style={{color:"white", backgroundColor:"transparent", marginBottom:".6vh"}}><Link className="links" to={"/cookiepolicy"} onClick={() => window.scrollTo(0, 0)} style={{fontSize:"20px"}}>Cookie Policy</Link></li>
+                <li style={{color:"white", backgroundColor:"transparent", marginBottom:".6vh"}}><Link className="links" to={"/privacypolicy"} onClick={() => window.scrollTo(0, 0)} style={{fontSize:"20px"}}>Privacy Policy</Link></li>
+                <li style={{color:"white", backgroundColor:"transparent", marginBottom:".6vh"}}><Link className="links" to={"/accessibilitypolicy"} onClick={() => window.scrollTo(0, 0)} style={{fontSize:"20px"}}>Accessibility Policy</Link></li>
```
Comment (BalajiSriraman) [inline Client/src/components/Footer/Footer.jsx:57]: adding style tag with similar properties can be avoided using common **class**

**Review by BalajiSriraman — APPROVED** [review]
LGTM

**Pull request merged**